sql/core/src/test/resources/sql-tests/inputs/pgSQL/numeric.sql

```diff
@@ -0,0 +1,1098 @@
+--
+-- Portions Copyright (c) 1996-2019, PostgreSQL Global Development Group
+--
+--
+-- NUMERIC
+-- https://github.com/postgres/postgres/blob/REL_12_BETA2/src/test/regress/sql/numeric.sql
+--
+-- This test suite contains eight Cartesian products without using explicit CROSS JOIN syntax.
+-- Thus, we set spark.sql.crossJoin.enabled to true.
+set spark.sql.crossJoin.enabled=true;
```

Comment:
PostgreSQL accepts cartesian joins by default. So, if the other ported tests include these joins, I think better to move this option into `SQLQueryTestSuite` to accept them for these PostgreSQL tests?